tools/kmt/kmt.c

```diff
@@ -1348,18 +1416,20 @@ chk_update(struct km_impl *impl, struct km_rec *r, bool reset_hash)
         hash_update(r);
 
     /* If you get a SIGBUS here during init ensure you have
-     * at least 16G of free space in /var/tmp.
+     * less than 16G of free space in /var/tmp.
```

Comment:
Confused.  Made more sense before: SIGBUS if there's not enough space

Reply:
Doh!  I read it wrong...

Reply:
Unfixed :-)